docs/tracking.md

```diff
@@ -1 +1,1058 @@
+
+Under the assumption of a homogeneous magnetic field along the $z$-axis,
+charged particles should follow helical trajectories. In the transverse plane,
+the motion is circular, while it is a straight line in the $rz$-plane.  The
+transverse impact parameter and momentum can be estimated from the radius of
+the circle in the transverse plane like
+
+$$
+  d_0 = \sqrt{c_x^2 + c_y^2} - \rho,
+$$
+
+with the circle center $(c_x, c_y)$ and radius $\rho$. The
+transverse momentum can be related to available quantities like
+
+$$
+  p_\mathrm{T} \propto \cdot q B \rho
+$$
+
+with the charge $q$ and the magnetic field $B$. An intersection
+between the straight line in the $rz$-plane with the $z$-axis gives an
+estimate of the longitudinal impact parameter. 
+An illustration of seeds in the transverse plane is found in
+{numref}`seeding`. Note that seeds can incorporate hits spread across all of
+the layers shown, although this can be a configuration parameter.
+
+
+(seeding)=
+:::{figure} /figures/tracking/seeding.svg
+:width: 300px
+:align: center
+Sketch of seeds in the transverse plane for a number of tracks on
+four layers. Seeds can combine hits on any three of these layers. The shown
+seeds appear compatible with having originated in the center of the
+detector, which is also drawn.
+:::
+
+## Track finding and track fitting
+
+In the track seeding and following approach, track candidates are built from
+the initial seeds. One method implemented in ACTS, the [Combinatorial Kalman
+Filter](#combinatorial-kalman-filter) (CKF), uses the *Kalman formalism*.
+Originally developed for monitoring and steering mechanical systems, it can
+also be used to iteratively calculate a track estimate. After a set of track
+candidates has been assembled and filtered (see [](#ambiguity-resolution)), an
+additional track fit is usually performed to extract the best estimate of the
+track. The Kalman formalism can also be used for this, with the addition of a
+smoothing step that has certain benefits.  Other fit strategies exist, such as
+a global $\chi^2$ fit that minimizes the distances between track-sensor
+intersections and measurements on all sensors at the same time. One drawback of
+this method is the necessity to invert very large matrices, which is
+computationally expensive.
+
+In a track fit, the Kalman formalism can be shown to yield optimal estimates
+for Gaussian uncertainties. This assumption breaks down when effects like
+bremsstrahlung come into play. An extension of the Kalman Filter (KF) exists
+that relies on the individual propagation of a set of trajectories, instead of
+a single one, to model these biased uncertainties by a sum of Gaussian
+components. This Gaussian Sum Filter (GSF) achieves better results when
+fitting particles such as electrons, likely to undergo bremsstrahlung, and is
+deployed in e.g. the ATLAS tracking chain.
+
+:::{attention}
+TODO: Add GSF link
+:::
```

Comment:
Did I understand correctly, what you inteded with the TODO?

```suggestion
components. This Gaussian Sum Filter [(GSF)](gsf)achieves better results when
fitting particles such as electrons, likely to undergo bremsstrahlung, and is
deployed in e.g. the ATLAS tracking chain.
```

Reply:
I actually wanted to keep this in to remind us. What do you think?

Reply:
Sure let's keep it.